server/routes/auth.js

```diff
@@ -0,0 +1,27 @@
+var router = require('express').Router();
+var sha256= require('sha256');
+var salt='10293018@!3$2%^';
+
+router.get('/signup', function(req, res){
+    res.render('signup.ejs');
+});
+
+router.post('/signup', function(req, res){
+     req.app.db.collection('user').insertOne({userId:req.body.id, email:req.body.email1+'@'+req.body.email2, password:sha256(req.body.password+salt), nickname:req.body.nickname}, 
```

Comment:
`req.body`를 한번에 넘기지 못한다면 가독성을 위해 행을 분리하거나 따로 변수를 선언해 정리한 후 넘기는 것이 좋아보이네요.
```suggestion
     req.app.db.collection('user').insertOne({
         userId:req.body.id, 
         email:req.body.email1+'@'+req.body.email2, 
         password:sha256(req.body.password+salt), 
         nickname:req.body.nickname
         }, 
```
or
```suggestion
     let userInfo = {
         userId:req.body.id, 
         email:req.body.email1+'@'+req.body.email2, 
         password:sha256(req.body.password+salt), 
         nickname:req.body.nickname
     };
     req.app.db.collection('user').insertOne(userInfo, 
```

Reply:
훨씬 깔끔하네요! 행 분리하는 방법으로 수정해보겠습니다.

Reply:
넵 제이쿼리로 통일했습니다!